gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -426,7 +426,7 @@ public default Control asControl() {
     }
   }
 
-  private static class BasicInputArea extends Composite implements InputArea {
+  public static class BasicInputArea extends Composite implements InputArea {
```

Comment:
Leave this one private, but make the GpuCountersDialog not be an inner class of this, but just of `TraceConfigDialog`.

Reply:
Thanks for suggestion, this organization would suit the current situation better!